workers/tasks/export/html.tsx

```diff
@@ -14,6 +14,17 @@ const nonExportableNodeTypes = ['discussion'];
 const katexCdnPrefix = 'https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.11.1/';
 const bullet = ' • ';
 
+const createCJKFontScript = `
```

Comment:
Could you add a comment to the effect of:

```
This script is provided by the "cjk-fonts" Web Fonts project that we manage from here:
https://fonts.adobe.com/my_fonts#web_projects-section
```
so that future readers know where it came from?

I might also call the variable `loadCjkFontsScript` since "load" feels like a more accurate verb than "create".